cprover_bindings/src/goto_program/expr.rs

```diff
@@ -111,6 +111,10 @@ pub enum ExprValue {
     },
     /// `__nondet()`
     Nondet,
+    /// Poison corresponds to a region of memory that
```

Comment:
Poison is more than uninitialized, right? Or are we narrowing down the concept for now?

I do wonder if we should state here that CBMC doesn't currently support poison and this gets translated to nondet.

It might be helpful to add a link to https://github.com/diffblue/cbmc/issues/7014.